zypp-plugin/snapper_zypp_plugin.cc

```diff
@@ -302,7 +302,6 @@ set<string> SnapperZyppPlugin::get_solvables(const Message& msg, Phase phase) {
 
     if (json_object_get_type(steps) == json_type_array) {
 	size_t i, len = json_object_array_length(steps);
-	printf("steps: %zu\n", len);
```

Comment:
maybe document in top of plugin that no such calls are allowes otherwise this can happen ( together with bsc) ?

Reply:
```
    sendback = sys.stdout
    sys.stdout = sys.stderr
    print "debug...."
    print >>sendback, "plugin responses..."
```
My py-plugins immediately redirect stdout to stderr after having made a copy of stdout which is explicitely used to send the plugin responses.

Reply:
Thanks!
Instead I made a forwarding-zypp-plugin helper that catches the bug at `make check` time.